src/variable.cpp

```diff
@@ -81,6 +91,9 @@ std::string Variable::doSerialisation(FORMATS format) const
         if (getInitialValue().length()) {
             repr += " initial_value=\"" + getInitialValue() + "\"";
         }
+        if (getInterfaceType() != Variable::INTERFACE_TYPES::INTERFACE_TYPE_UNDEFINED) {
```

Comment:
Just for clarity, I believe the above discussion resolves this as being fine to not serialise the case when the interface is `none`.